MvvmCross.Android.Support/V7.AppCompat/MvxAppCompatActivity.cs

```diff
@@ -87,6 +87,8 @@ protected override void OnCreate(Bundle bundle)
         {
             base.OnCreate(bundle);
             ViewModel?.ViewCreated();
+
+            //TODO: Add setup init and startup here
```

Comment:
What does this comment mean here? Is that needed to enable android apps start without splash screens, right?

Reply:
@martijn00 did you need me to add the setup logic, or are you going to add this before we merge this PR?

Reply:
Yeah it's pretty late here, would be good if you have time. Should be a matter of copying some logic in here and it should work.